readme.md

```diff
@@ -0,0 +1,106 @@
+# <div align="center"> Express Rate Limit - Memcached Store </div>
+
+<div align="center">
+
+[![tests](https://github.com/express-rate-limit/rate-limit-memcached/actions/workflows/ci.yaml/badge.svg)](https://github.com/express-rate-limit/rate-limit-memcached/actions/workflows/ci.yaml)
+[![npm version](https://img.shields.io/npm/v/rate-limit-memcached.svg)](https://npmjs.org/package/rate-limit-memcached 'View this project on NPM')
+[![npm downloads](https://img.shields.io/npm/dm/rate-limit-memcached)](https://www.npmjs.com/package/rate-limit-memcached)
+
+A memcached store for the
+[express-rate-limit](https://github.com/express-rate-limit/express-rate-limit)
+middleware.
+
+</div>
+
+## Installation
+
+From the npm registry:
+
+```sh
+# Using npm
+> npm install express-rate-limit
+# Using yarn or pnpm
+> yarn/pnpm add express-rate-limit
+```
+
+From Github Releases:
+
+```sh
+# Using npm
+> npm install https://github.com/express-rate-limit/rate-limit-memcached/releases/download/v{version}/rate-limit-memcached.tgz
+# Using yarn or pnpm
+> yarn/pnpm add https://github.com/express-rate-limit/rate-limit-memcached/releases/download/v{version}/rate-limit-memcached.tgz
+```
+
+Replace `{version}` with the version of the package that you want to your, e.g.:
+`2.0.0`.
+
+## Usage
+
+> This package, from version `2.0.0` onwards, is a
+> [pure `esm` package](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c).
+> This means you cannot `require` it in `cjs` projects anymore. Please see the
+> linked article for more information.
+
+**This package requires you to use Node 16 or above.**
+
+An example of its usage is as follows:
+
+```ts
+import { rateLimit } from 'express-rate-limit'
+import { MemcachedStore } from 'rate-limit-memcached'
+
+const limiter = rateLimit({
+	windowMs: 15 * 60 * 1000, // 15 minutes.
+	max: 100, // Limit each IP to 100 requests per `window` (here, per 15 minutes).
+	standardHeaders: 'draft-7', // Return rate limit info in the `RateLimit` header.
+	legacyHeaders: false, // Disable the `X-RateLimit-*` headers.
+	store: new MemcachedStore({
+		// prefix: 'rl:', // The prefix attached to all keys stored in the cache.
+		// client: new Memcached(['foo.bar', 'localhost:11211']) // The memcached client to use.
```

Comment:
If we add support for passing in a locations param, then the example here can look like 
```suggestion
		locations: ['localhost:11211'], // server location(s), passed directly to Memcached
```